packages/astro/package.json

```diff
@@ -4,6 +4,13 @@
   "description": "Official Sentry SDK for Astro",
   "repository": "git://github.com/getsentry/sentry-javascript.git",
   "homepage": "https://github.com/getsentry/sentry-javascript/tree/master/packages/astro",
+  "keywords": [
+    "withastro",
+    "astro-component",
+    "analytics",
```

Comment:
Talked to Astro folks. They'll add a new category sometime next week. For now, we remove `analytics`. Can add the new category later on. 

```suggestion
```